src/brouwers/kitreviews/models.py

```diff
@@ -71,23 +74,16 @@ def topic_url(self):
         if self.topic_id:
```

Comment:
No, it's an implementation detail that I'm not proud of :(

It works similarly to ForeignKey, but the other way around. If you define `foo = models.ForeignKey(...)`, the model will have a `foo_id` property. Here, `topic_id = ForumToolsIDField(...)` will take care of ensuring that the `topic` prop is available. This is some sort of cross-database foreignkey (without the Integrity checks then).